firmware/targets/f7/api_symbols.csv

```diff
@@ -1,5 +1,5 @@
 entry,status,name,type,params
-Version,+,10.1,,
+Version,+,13.1,,
```

Comment:
No need to increase the major version, the minor version is enough.

Reply:
Actually there is a need. speaker api now behaves differently.